src/Service/Startup.cs

```diff
@@ -189,7 +190,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IAuthorizationResolver, AuthorizationResolver>();
             services.AddSingleton<IOpenApiDocumentor, OpenApiDocumentor>();
 
-            AddGraphQLService(services);
+            int? depthLimit = null;
+            if (runtimeConfig is not null
+                && runtimeConfig.Runtime!.GraphQL!.DepthLimit is not null
+                && runtimeConfig.Runtime!.GraphQL!.DepthLimit > 0)
```

Comment:
```suggestion
                && runtimeConfig.Runtime!.GraphQL!.DepthLimit is not null
                && runtimeConfig.Runtime!.GraphQL!.DepthLimit > 0)
```

all `!` can be changed to `?`